modules/efk-logging-elasticsearch-exposing.adoc

```diff
@@ -96,14 +88,18 @@ route.route.openshift.io/elasticsearch created
 .. Get the token of this ServiceAccount to be used in the request:
 +
 ----
-$ token=$(oc whoami -t)
+$ routeES=`oc get route elasticsearch -o jsonpath={.spec.host}`
 ----
 
-.. Run a command similar to the following, using your cluster address to access Elasticsearch through the exposed route:
+.. Run the following command to access Elasticsearch through the exposed route:
++
+----
+curl -tlsv1.2 -v --insecure -H "Authorization: Bearer ${token}" "https://${routeES}/.operations.*/_search?size=1" | jq
```

Comment:
Could you remove "-v"?  It is a switch to enable the verbose mode.  It's not needed for most users. :)